src/test/java/org/opensearch/sdk/TestExtensionInterfaces.java

```diff
@@ -150,4 +147,63 @@ void testSystemIndexExtension() {
         };
         assertTrue(systemIndexExtension.getSystemIndexDescriptors(null).isEmpty());
     }
+
+    @Test
+    void getRepositoriesReturnsEmptyMapByDefault() {
+
+        Environment env = mock(Environment.class);
+        NamedXContentRegistry namedXContentRegistry = mock(NamedXContentRegistry.class);
+        ClusterService clusterService = mock(ClusterService.class);
+        RecoverySettings recoverySettings = mock(RecoverySettings.class);
+
+        RepositoryExtension repositoryExtension = new RepositoryExtension() {
+            @Override
+            public Map<String, Repository.Factory> getRepositories(
+                Environment env,
+                NamedXContentRegistry namedXContentRegistry,
+                ClusterService clusterService,
+                RecoverySettings recoverySettings
+            ) {
+                return RepositoryExtension.super.getRepositories(env, namedXContentRegistry, clusterService, recoverySettings);
+            }
+
+            @Override
+            public Map<String, Repository.Factory> getInternalRepositories(
+                Environment env,
+                NamedXContentRegistry namedXContentRegistry,
+                ClusterService clusterService,
+                RecoverySettings recoverySettings
+            ) {
+                return RepositoryExtension.super.getInternalRepositories(env, namedXContentRegistry, clusterService, recoverySettings);
+            }
+        };
+
+        Map<String, Repository.Factory> repositories = repositoryExtension.getRepositories(
+            env,
+            namedXContentRegistry,
+            clusterService,
+            recoverySettings
+        );
+        assertEquals(Collections.emptyMap(), repositories);
+    }
+
+    @Test
+    void getInternalRepositoriesReturnsEmptyMapByDefault() {
```

Comment:
I know we have a mix of extension interface tests vs. extension point tests. I'd prefer to name this method `testRepositoryExtension()`